confirmation_rule.py

```diff
@@ -180,6 +181,81 @@ def is_one_confirmed(store: Store, block_root: Root) -> bool:
     )
 
 
+def get_committee_at_slot(state: BeaconState, slot: Slot) -> Sequence[ValidatorIndex]:
+    indices = []
+    committees_count = get_committee_count_per_slot(state, compute_epoch_at_slot(state.slot))
+    for i in range(committees_count):
+        indices.append(get_beacon_committee(state, Slot(slot), CommitteeIndex(i)))
+    return indices
+
+
+def compute_chain_prefix_support_between_slots(
+        store: Store, block_root: Root, balance_source: BeaconState, first_slot: Slot, last_slot: Slot) -> Gwei:
+    head = get_head(store)
+    head_state = store.block_states[head]
+    support = Gwei(0)
+    for slot in range(first_slot, last_slot + 1):
+        committee = get_committee_at_slot(head_state, get_block_slot(store, block_root))
+        for validator_index in committee:
+            if (store.latest_messages[validator_index].root == block_root
+                and validator_index not in store.equivocating_indices):
+                support += balance_source.validators[validator_index].effective_balance
+
+    return support
+
+
+def compute_honest_parent_support(store: Store, block_root: Root, weighting_checkpoint_state: State) -> Gwei:
+    block = store.blocks[block_root]
+    parent_block = store.blocks[block.parent_root]
+    if parent_block.slot + 1 == block.slot:
+        parent_support = Gwei(0)
+        parent_maximum_support = Gwei(0)
+    else:
+        parent_support = compute_chain_prefix_support_between_slots(
+            store, block_root.parent_root, weighting_checkpoint_state, Slot(parent_block.slot + 1), Slot(block.slot - 1))
+        parent_maximum_support = estimate_committee_weight_between_slots(
+            weighting_checkpoint_state, Slot(parent_block.slot + 1), Slot(block.slot - 1))
+
+    parent_support_inclusive = parent_support + compute_chain_prefix_support_between_slots(
+        store, block_root.parent_root, weighting_checkpoint_state, Slot(block.slot), Slot(block.slot))
```

Comment:
```suggestion
    parent_support_inclusive = compute_chain_prefix_support_between_slots(
        store, block_root.parent_root, weighting_checkpoint_state, Slot(parent_block.slot + 1), Slot(block.slot))
```

nit: maybe do it uniformly for the sake of clarity? Implementations will optimize